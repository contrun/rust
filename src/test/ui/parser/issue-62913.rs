"\u\\"
//~^ ERROR incorrect unicode escape sequence
//~| ERROR invalid trailing slash in literal
//~| ERROR statements cannot reside in modules
//~| ERROR `main` function not found
