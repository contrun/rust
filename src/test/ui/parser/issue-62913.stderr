error: incorrect unicode escape sequence
  --> $DIR/issue-62913.rs:1:2
   |
LL | "\u\"
   |  ^^^ incorrect unicode escape sequence
   |
   = help: format of unicode escape sequences is `\u{...}`

error: invalid trailing slash in literal
  --> $DIR/issue-62913.rs:1:5
   |
LL | "\u\"
   |     ^

error: statements cannot reside in modules
  --> $DIR/issue-62913.rs:1:1
   |
LL | "\u\"
   | ^^^^^^
   |
   = note: the program entry point starts in `fn main() { ... }`, defined in `main.rs`
   = note: for more on functions and how to structure your program, see https://doc.rust-lang.org/book/ch03-03-how-functions-work.html
help: consider moving the statements into a function
   |
LL | fn my_function() -> _ { "\u\" }
   |

error[E0601]: `main` function not found in crate `issue_62913`
  --> $DIR/issue-62913.rs:1:1
   |
LL | "\u\"
   | ^^^^^^ consider adding a `main` function to `$DIR/issue-62913.rs`

error: aborting due to 4 previous errors

For more information about this error, try `rustc --explain E0601`.
